Formula/libvisual-0.4.rb

```diff
@@ -0,0 +1,23 @@
+class Libvisual04 < Formula
+  desc "Audio Visualization"
+  homepage "https://github.com/Libvisual/libvisual"
+  url "https://github.com/Libvisual/libvisual/releases/download/libvisual-0.4.2/libvisual-0.4.2.tar.gz"
+  sha256 "63085fd9835c42c9399ea6bb13a7ebd4b1547ace75c4595ce8e9759512bd998a"
+  license "LGPL-2.1-or-later"
+  # head "https://github.com/Libvisual/libvisual", branch: "0.4.x"
+
+  depends_on "pkg-config" => :build
+  depends_on "sdl12-compat"
+
+  def install
+    args = ["--prefix=#{prefix}"]
+    system "./configure", *args
+    system "make", "-j2"
```

Comment:
```suggestion
    system "make"
```
`brew` already handles parallelisation.

Reply:
@carlocab I see. Do you have a link on details where I could learn more about it? Does `system` magically check for invokcation of `make`?

Reply:
Nothing that complicated -- `brew` just sets `MAKEFLAGS` to `-jX` where `X` is the number of available cores. (But this can be overridden by `HOMEBREW_MAKE_JOBS`.)

Reply:
@carlocab I see, thanks! :+1: